### PR TITLE
modal added

### PR DESCRIPTION
**Pull request by cheedinmar**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     },
     "dependencies": {
         "@fortawesome/fontawesome-free": "5.14.0",
+        "@linusborg/vue-simple-portal": "^0.1.5",
```
Comment (AltaraNg) [inline package.json:15]: what does this package do?

File: src/components/Cards/calculator.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class=" lg:w-2/5 w-full  bg-white lg:flex-col pb-16  rounded lg:my-20 my-0 lg:mx-20 mx-0 h-full  hover:shadow-2xl">
+          <h3 class=" tracking-wider mb-32 leading-6 text-2xl font-bold text-center px-8 mt-16 text-black  ">
+                   Your initial payment is <br><p class="underline pt-8"> ₦20,900.00</p>
```
Comment (AltaraNg) [inline src/components/Cards/calculator.vue:4]: why do we have hardcoded figures?

File: src/views/calculator/index.vue [hunk 3]
```diff
@@ -73,6 +74,14 @@ export default {
...
            
        }
-   } 
+   } ,
+   methods:{
+    toggleModal(){
+      this.modal =!this.modal;
+    },
+     onModalClose (){
+      modalState.value = false
```
Comment (AltaraNg) [inline src/views/calculator/index.vue:83]: which variable is `modalState `?

File: src/components/Cards/calculator.vue (hunk 2 above)
Comment (cheedinmar) [reply src/components/Cards/calculator.vue:4]: I just used it ti design the UI, the data will be fetched... I am not done with it

File: src/views/calculator/index.vue (hunk 3 above)
Comment (cheedinmar) [reply src/views/calculator/index.vue:83]: A mistake, I will get it updated on the next PR

File: package.json (hunk 1 above)
Comment (cheedinmar) [reply package.json:15]: A Portal Component for Vuejs, to render DOM outside of a component, anywhere in the document.

**Pull request closed without merging**